src/Support/SelectFields.php

```diff
@@ -192,7 +192,8 @@ protected static function handleFields(
                     );
                 }
                 // With
-                elseif (is_array($field['fields']) && $queryable) {
+
+                elseif (is_array($field['fields']) && !empty($field['fields']) && $queryable) {
```

Comment:
I see that lots-o-tests are failing without the empty check, but failed to properly understand: why, now?

Reply:
it returns **fields** for simple properties also:

![image](https://user-images.githubusercontent.com/3076578/121263046-5a075d80-c8b5-11eb-8fbe-efe0457b5a7f.png)


!empty might actually be enough here, will check